src/components/ImageZoom/ImageZoom.tsx

```diff
@@ -0,0 +1,136 @@
+'use client';
+
+import classNames from 'classnames/bind';
+import { useState, MouseEvent, useRef, SyntheticEvent } from 'react';
+import Image, { StaticImageData } from 'next/image';
+
+import { IMAGE_BLUR } from '@/constants/blurImage';
+import { keydeukProfileImg } from '@/public/index';
+import ZoomView from './ZoomView';
+
+import styles from './ImageZoom.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface ImageZoomProps {
+  image: string | StaticImageData;
+  alt: string;
+  width: number;
+  height: number;
+}
```

Comment:
```suggestion
interface ImageZoomProps extends ComponentPropsWithoutRef<typeof Image> {
  width: number;
  height: number;
}
```
이거도 그냥 참고만!! 최근에 `ComponentPropsWithoutRef`에 대해 배워서 공유 합니당~! 이렇게 하면 이미지의 기본속성타입을  가져올 수 있어용!

Reply:
오ㅗ오ㅗ옹옹 완젼 꿀팁,,,,